### DISCUSSION TITLE
Roles and Resources

### DISCUSSION THREAD

**Discussion by jeff-zucker** [solid-contrib/practitioners#2] [Ideas]
I volunteer for the following unless someone objects - facilitate/moderate calls and chat rooms;  create/manage github repo(s); work on developing RDF-based forms (collaborators wanted);  work on generating website & social media content (collaborators wanted).

Functions/Resources it would be nice to have eventually
* resources to carry out a concerted outreach effort with emphasis on diversity
* people interested in mentoring
* finances for paid internships and mentoring
* people to manage event invitations, website & social media posting

**Comment by timea-solid** [comment]
For now I would focus on "facilitate/moderate calls and chat rooms" and listen to the needs of the participants what they would need. A website and social media could be possible needs but I would not jump on those to start with.
